crates/sqlbuiltins/src/functions/lance.rs

```diff
@@ -0,0 +1,47 @@
+use std::collections::HashMap;
+use std::sync::Arc;
+
+use crate::functions::table_location_and_opts;
+use async_trait::async_trait;
+use datafusion::datasource::TableProvider;
+use datafusion_ext::errors::{ExtensionError, Result};
+use datafusion_ext::functions::{FuncParamValue, TableFunc, TableFuncContextProvider};
+use datasources::lance::scan_lance_table;
+use protogen::metastore::types::catalog::RuntimePreference;
+
+/// Function for scanning delta tables.
+///
+/// Note that this is separate from the other object store functions since
+/// initializing object storage happens within the deltalake lib. We're
```

Comment:
```suggestion
/// initializing object storage happens within the lance lib. We're
```

Probably, idk how it's actually created.